### PR TITLE
doc: split classes into files

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -1,26 +1,17 @@
-<!-- THIS FILE IS NOW GENERATED -->
+---
+id: ci
```
Comment (dgozman) [inline docs/cli.md:2]: Wrong id and title here.

File: docs-src/actionability.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# Actionability
+---
+id: actionability
```
Comment (dgozman) [inline docs-src/actionability.md:2]: It seems like id is always the same as the file name. Do we need two? Can they differ?

**Review by dgozman — APPROVED** [review]

File: docs-src/actionability.md (hunk 2 above)
Comment (pavelfeldman) [reply docs-src/actionability.md:2]: Dunno, that's what Arjun generates!

Comment (pavelfeldman) [reply docs-src/actionability.md:2]: Dunno, that's what Arjun geberates!

**Pull request merged**